#Use target directory instead of copying files to cups dir.

diff -Naur splix-1.1.0/Makefile splix-1.1.0.new/Makefile
--- splix-1.1.0/Makefile	2007-11-16 03:26:11.000000000 +0100
+++ splix-1.1.0.new/Makefile	2008-01-26 23:28:39.000000000 +0100
@@ -6,8 +6,10 @@
 
 CXXFLAGS	:= -O2 `cups-config --cflags` 
 LDFLAGS	:= `cups-config --ldflags`
-CUPSFILTER	:= `cups-config --serverbin`/filter
-CUPSPPD	:= `cups-config --datadir`/model
+#CUPSFILTER	:= `cups-config --serverbin`/filter
+CUPSFILTER	:= @%Compile_target%@/lib/cups/filter
+#CUPSPPD	:= `cups-config --datadir`/model
+CUPSPPD	:= @%Compile_target%@/Shared/cups/model
 
 # === DON'T CHANGE ANYTHING AFTER THIS MESSAGE ====
 
